dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/oozie/AbstractOozieJobsTask.java

```diff
@@ -70,7 +72,7 @@ protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Ha
     try (CSVPrinter printer = csvFormat.print(sink.asCharSink(UTF_8).openBufferedStream())) {
       XOozieClient oozieClient = ((OozieHandle) handle).getOozieClient();
       final int batchSize = context.getArguments().getPaginationPageSize();
-      int offset = 1; // starts with 1, not 0.
+      int offset = INITIAL_OOZIE_JOBS_OFFSET;
```

Comment:
If the constant is used only here,
I personally prefer the previous version:
```int offset = 1; // starts with 1, not 0.```
but I am almost sure somebody requested this change as part of a code review :D